Spigot-Server-Patches/0399-Implement-alternative-item-despawn-rate.patch

```diff
@@ -0,0 +1,82 @@
+From 5782fc0ff56779a77b4a1025aaf1dfc8d97939a6 Mon Sep 17 00:00:00 2001
+From: kickash32 <kickash32@gmail.com>
+Date: Mon, 3 Jun 2019 02:02:39 -0400
+Subject: [PATCH] Implement alternative item-despawn-rate
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+index b8540619..d6c489e0 100644
+--- a/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
++++ b/src/main/java/com/destroystokyo/paper/PaperWorldConfig.java
+@@ -1,9 +1,15 @@
+ package com.destroystokyo.paper;
+ 
++import java.util.ArrayList;
++import java.util.Arrays;
++import java.util.HashSet;
+ import java.util.List;
++import java.util.Set;
+ 
+ import org.bukkit.Bukkit;
++import org.bukkit.Material;
+ import org.bukkit.configuration.file.YamlConfiguration;
++import org.bukkit.entity.EntityType;
+ import org.spigotmc.SpigotWorldConfig;
+ 
+ import static com.destroystokyo.paper.PaperConfig.log;
+@@ -109,6 +115,29 @@ public class PaperWorldConfig {
+         hardDespawnDistance = hardDespawnDistance*hardDespawnDistance;
+     }
+ 
++    public boolean altItemDespawnRateEnabled;
++    public Set<Material> altItemDespawnRateItems;
++    public int altItemDespawnRate;
++    private void altItemDespawnRate(){
++        altItemDespawnRateEnabled = getBoolean("alt-item-despawn-rate.enabled", false);
++        System.out.println(altItemDespawnRateEnabled);
++        altItemDespawnRateItems = new HashSet<>();
++        for(Object input : getList("alt-item-despawn-rate.items", Arrays.asList("COBBLESTONE","STONE","DIRT","SANDSNOWBALL","NETHERRACK"))){
```

Comment:
~not sure if we want to supply default values like that, might be confusing to ppl when they update~
nvm, its disabled by default.

Reply:
I don't like the Object var here, getList is generic, you should be able to make java use the string type here.